api/data_explorer/controllers/facets_controller.py

```diff
@@ -18,8 +20,20 @@ def facets_get():
     response = search.execute()
     facets = []
     for facet_name, values in response.facets.to_dict().iteritems():
+        # isinstance(current_app.config['ELASTICSEARCH_FACETS']['Age'], HistogramFacet)
+        # hf._params['interval']
         facet_values = []
         for name, count, _ in values:
-            facet_values.append(FacetValue(name=name, count=count))
+            es_facet = current_app.config['ELASTICSEARCH_FACETS'][facet_name]
+            if isinstance(es_facet, HistogramFacet):
+              # For histograms, Elasticsearch returns:
+              #   name 10: count 15     (There are 15 people aged 10-19)
+              #   name 20: count 33     (There are 33 people aged 20-29)
+              # Convert "10" -> "10-19".
+              range_str = '%d-%d' % (name,
+                  name + es_facet._params['interval'] - 1)
```

Comment:
The assumption here being that numeric ranges are always in discreet whole numbers. Is this always the case? i.e. if my intervals are 0 - 0.1, 0.1 - 0.2, we wouldn't want to apply -1.

Reply:
Makes sense. Let's address this when we come across this.